docs/en/contribution/contribute-plugin.md

```diff
@@ -140,3 +140,109 @@ start a Clickhouse database in your test. You can use `GenericContainer` to star
 
 It should be noted that your e2e test class should be named ending with `IT`. By default, we will not execute the test if the class name ending with `IT`.
 You can add `-DskipIT=false` to execute the e2e test, it will rely on a Docker environment.
+
+# Connector-v2 Contribution Guide
+
+## Purpose
+
+The following describes the new interface and the new code structure on account of the newly designed API for Connectors in Apache SeaTunnel. This helps developers with quick overview regarding API, translation layer improvement, and development of new Connector.
+
+## Code Structure
+
+In order to separate from the old code, we have defined new modules for execution flow. This facilitates parallel development at the current stage, and reduces the difficulty of merging. All the relevant code at this stage is kept on the ``api-draft`` branch.
+
+### Example
+
+We have prepared a new version of the locally executable example program in ``seatunnel-examples``, which can be directly called using ``seatunnel-flink-connector-v2-example`` or ``seatunnel-spark-connector-v2-example`` in ``SeaTunnelApiExample``. This is also the debugging method that is often used in the local development of Connector. The corresponding configuration files are saved in the same module ``resources/examples`` folder as before.
+
+
+### Startup Class
+
+Aside from the old startup class, we have created two new startup class projects, namely ``seatunnel-core/seatunnel-flink-starter`` and ``seatunnel-core/seatunnel-spark-starter``. You can find out how to parse the configuration file into an executable Flink/Spark process here.
+
+### SeaTunnel API
+
+A new ``seatunnel-api`` (not ``seatunnel-apis``) module has been created to store the new interfaces defined by the SeaTunnel API. By implementing these interfaces, developers can complete the SeaTunnel Connector that supports multiple engines.
+
+### Translation Layer
+
+We realize the conversion between SeaTunnel API and Engine API by adapting the interfaces of different engines, so as to achieve the effect of translation, and let our SeaTunnel Connector support the operation of multiple different engines. The corresponding code address, ``seatunnel-translation``, this module has the corresponding translation layer implementation. If you are interested, you can view the code and help us improve the current code.
+
+## API introduction
+
+The API design of the current version of SeaTunnel draws on the design concept of Flink.
+
+### Source
+
+#### SeaTunnelSource.java
```

Comment:
link to connector-v2-readme?

https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-connectors-v2

Reply:
Yes, it has been modified to link to connector-v2-readme